mautic/mautic
# Bug clear scripts from notifications

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4084
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR fixes the issue that the Mautic UI can be unaccessible if a notification contains some `<script/>` tag.

This PR is replacement of https://github.com/mautic/mautic/pull/4185 since that PR was fixing it only for one plugin. This PR fixes it for whatever goes into the notifications.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to notifications table and hack some active notification there. Add `<script>alert('script is running');</script><SCRIPT>alert('CAPITAL script is running');</SCRIPT>` into the message.
2. Refresh the page and you should get the JS alerts.

#### Steps to test this PR:
1. Apply this PR
2. Refresh Mautic and you should not get the alerts.
